util/make_new_dif/dif_autogen.c.tpl

```diff
@@ -208,3 +210,5 @@ dif_result_t dif_${ip.name_snake}_irq_restore_all(
 
   return kDifOk;
 }
+
+% endif
```

Comment:
We don't really have good guidance on this... I'd really like it if there was a way to indicate which `if` this matched up with. =/

We can add silly indentation to the file... or something? Not sure. What are your thoughts?

Reply:
That works for me! Fixed.

Reply:
Indenting C snippet within mako constructs end up showing up in the generated code, which looks unnecessarily indented. Perhaps revert this back?

Reply:
I see - `clang-format` fixes it. Unfortunately we do not have a formatter for other autogenerated code (SV, etc). Perhaps we should hold off on this until we come up with one?